interchain-exchange/tutorial/00-tutorial.md

```diff
@@ -388,6 +408,51 @@ balances:
 
 The counterparty (sender of the buy mcx order) received 5 mcx vouchers. vcx balance hasn't changed, because the correct amount of vcx (50) were locked at the creation of the buy order during the previous step.
 
+```yml
+# Target blockchain
+balances:
+- amount: "5" # increased from 0
+  denom: ibc/99678A10AF684E33E88959727F2455AE42CCC64CD76ECFA9691E1B5A32342D33 # mcx voucher
+```
+-->
+
+We now have two orders open for `marscoin`: a *sell order* on the **source chain** (for 20 `marscoin` at 5 `venuscoin`) and a *buy order* on the **target chain** (for 5 `marscoin` at 5 `venuscoin`). Let's perform an exchange by sending a sell order to the source chain.
+
+```
+# Sell order broadcasted to the source chain
+interchanged tx ibcdex send-sellOrder ibcdex channel-0 mcx 20 vcx 5
+```
+
+The sell order (for 5 `marscoin` at 5 `venuscoin`) was filled on the target chain by the buy order. The amount of the buy order on the target chain has decreased by 5 `marscoin`.
+
+```yml
+# Target blockchain
+BuyOrderBook:
+- amountDenom: mcx
+  creator: ""
+  index: ibcdex-channel-0-mcx-vcx
+  orderIDTrack: 5
+  orders:
+  - amount: 15 # decreased from 20
+    creator: cosmos1qlrz3peenc6s3xjv9k97e8ef72nk3qn3a0xax2
+    id: 3
+    price: 5
+  priceDenom: vcx
+```
+
+The sender of the filled sell order exchanged 5 `marscoin` for 25 `venuscoin` vouchers. 25 vouchers is a product of the amount of the sell order (5 `marscoin`) and price of the buy order (5 `venuscoin`).
+
+```yml
+# Source blockchain
+balances:
+- amount: "25" # increased from 0
+  denom: ibc/50D70B7748FB8AA69F09114EC9E5615C39E07381FE80E628A1AF63A6F5C79833 # vcx voucher
+- amount: "985" # decreased from 990
+  denom: mcx
+```
+
+The counterparty (sender of the buy mcx order) received 5 `marscoin` vouchers. `venuscoin` balance hasn't changed, because the correct amount of `venuscoin` (50) were locked at the creation of the buy order during the previous step.
```

Comment:
do we mean mcx here? Or did we decide to spell out marscoin?

Reply:
I suggest to keep `marscoin` and `venuscoin` when explaining. For code we can use mcx and vcx

Reply:
oooh I know it's work, but I'm in agreement with Denis on this... is there a way to use consistent token names and denom? Teach me!